build/patches/Timezone-customization.patch

```diff
@@ -1287,7 +1287,8 @@ diff --git a/components/content_settings/renderer/content_settings_agent_impl.cc
 +  } else if (setting == CONTENT_SETTING_BLOCK) {
 +    // timezone random
 +    UErrorCode ec = U_ZERO_ERROR;
-+    int32_t rawOffset = ((rand() % 24) - 12) * 3600 * 1000;
++    std::srand(std::time(nullptr));
```

Comment:
It is a bad practice to do this every single time a random number is used. Please use `base::RandInt` instead

Reply:
fix in https://github.com/bromite/bromite/pull/830/commits/76992d379bc632413a9ec828457715737642b208